es2015_exercise/readme.md

```diff
@@ -11,23 +11,36 @@ var person = {
         }.bind(this),1000)
     }
 }
+var person2 = {
+    fullName: "Harry Potter",
+    sayHi: function(){
```

Comment:
this can be simplified more:

```
var person2 = {
  fullName: "Harry Potter",
  sayHi() {
    setTimeout( () => console.log(`Your name is ${this.fullName}`), 1000)
  }
}
```